blog/models.py

```diff
@@ -59,3 +59,7 @@ def approve(self):
     def __str__(self):
         """Represent a comment as a string by its text."""
         return self.text
+
+    def get_absolute_url(self):
+        """Redirect to view with detail info about parent post."""
+        return reverse('post_detail', args=[str(self.post.pk)])
```

Comment:
I think, `pk=self.post.pk` should work just fine

Reply:
It doesn't, reverse() accepts only args / kwargs for additional parameters.

Reply:
oh then you should pass kwargs, singe you use named group `pk` in that regexp